catalog/app/components/Preview/types.js

```diff
@@ -34,11 +35,11 @@ export const PreviewData = tagged([
   'Notebook', // { preview: string, ...PreviewStatus }
   'Parquet', // { preview: string, ...ParquetMeta, ...PreviewStatus }
   'Pdf', // { handle: object, pages: number, firstPageBlob: Blob, type: 'pdf' | 'pptx' }
+  'Perspective', // { preview: string, ...PreviewStatus }
```

Comment:
seems like type description is incorrect, should be smth like `{ context: ..., data: ..., handle: ..., ... }`